src/components/DropdownMultiple/itemList.tsx

```diff
@@ -75,7 +76,12 @@ const ItemComponent = React.memo<ItemProps>(props => {
                 }
                 size="24px"
             />
-            <Text>{props.item.text}</Text>
+            <Text>
+                {props.item.text}
+                {props.item.remarks === '' ? (
+                    <RemarksText>{props.item.remarks}</RemarksText>
+                ) : null}
```

Comment:
```suggestion
                {props.item.remarks === '' && <RemarksText>{props.item.remarks}</RemarksText>}
```

って書けたりしますかね？
これは好みの問題なんでどっちでもOKです

Reply:
こっちの方がスッキリしてますね。修正します！